olga/analytics/views.py

```diff
@@ -48,32 +121,66 @@ def update_students_without_country_value(active_students_amount, students_per_c
             students_per_country (dict): Country-count accordance as pair of key-value.
                                          Amount of students without country has calculated and inserted to
                                          corresponding key ('null').
-        """
 
+        """
         students_per_country['null'] = active_students_amount - sum(students_per_country.values())
 
         return students_per_country
 
-    def create_instance_data(self, received_data, secret_token):  # pylint: disable=too-many-locals
+    def extend_statistics_to_enthusiast_level(self, received_data, installation_statistics, edx_installation_object):
+        # pylint: disable=invalid-name
+        """
+        Extend installation statistics level from `Paranoid` to `Enthusiast`.
```

Comment:
Please elaborate on this method. At the moment it's not obvious what use cases does it cover.